### ISSUE TITLE
Consider organizing into folders and renaming

### ISSUE THREAD

**Issue by mwvaughn** [jetstream-cloud/cloudinit-userdata#1]
In keeping with the discussions we've been having around the APPS project, can I suggest we create a directory for each featured image. This lets us add documentation, additional assets like ansible playbooks or config scripts/files, etc. Can I also suggest renaming this repo from cloudinit-userdata to featured_images?

**Comment by jmlowe** [comment]
Yes, please.  There was very little if any thought put into the organization of this repo.  That needs to be corrected.
